tests/utils.go

```diff
@@ -2302,7 +2302,17 @@ func NewRandomFedoraVMIWithVirtWhatCpuidHelper() *v1.VirtualMachineInstance {
 	return vmi
 }
 
-func GetGuestAgentUserData() string {
+func GetFedoraToolsGuestAgentUserData() string {
+	return `#!/bin/bash
+            echo "fedora" |passwd fedora --stdin
+            sudo setenforce Permissive
+	    sudo mv /home/fedora/qemu-guest-agent.service /lib/systemd/system/
```

Comment:
please consider that after reboot the file wont be there
even if cloud-init wont stop on error, its better imo to add `[ -f file ] &&` ?

Reply:
yup, good point